torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3923,6 +3939,16 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_inplace_autograd=True,
            safe_casts_outputs=True,
            sample_inputs_func=sample_inputs_xlogy),
+    OpInfo('logsumexp',
+           dtypes=floating_types_and(torch.bfloat16),
+           dtypesIfCUDA=all_types_and(torch.bfloat16),
```

Comment:
As mentioned above, it looks like the op also supported torch.float16 (AKA torch.half) on CUDA

Note that torch.bfloat16 is not torch.float16